DataManagementSystem/DB/FileCatalogComponents/DirectoryMetadata.py

```diff
@@ -604,9 +626,9 @@ def findDirIDsByMetadata(self, queryDict, path, credDict):
       first = True
       for meta, value in finalMetaDict.items():
         if value == "Missing":
-          result = self.__findSubdirMissingMeta(meta, pathSelection)
+          result = self.__findSubdirMissingMeta(fqmeta, pathSelection)
```

Comment:
Should this still be `meta`, given that this comes from the loop variable two lines above?

Reply:
I think we need a line:
```python
fqmeta = self.getMetaName(meta, credDict)
```
as a first line in both loops, so ```python __findSubdirByMeta ``` looks into a correct table.